pkg/oras/oras_test.go

```diff
@@ -160,7 +161,7 @@ func (suite *ORASTestSuite) Test_1_Pull() {
 	// Pull chart-dir
 	store = orascontent.NewMemoryStore()
 	ref = fmt.Sprintf("%s/chart-dir:test", suite.DockerRegistryHost)
-	descriptors, err = Pull(newContext(), newResolver(), ref, store)
+	_, descriptors, err = Pull(newContext(), newResolver(), ref, store)
```

Comment:
Do you have a test for `WithAllowedMediaType` ?

Reply:
will add one

Reply:
added test `Test_2_MediaType`